src/TextButton/textButtonManifest.json

```diff
@@ -35,6 +36,23 @@
       "type": "icon",
       "default": "add"
     },
+    {
+      "name": "sizing",
+      "displayName": "Text & Icon Sizing",
+      "type": "number",
+      "default": 3,
+      "control": {
+        "type": "menu",
+        "options": [
+          { "label": "Gigantic", "value": 0 },
+          { "label": "Extra large", "value": 1 },
+          { "label": "Large", "value": 2 },
+          { "label": "Medium", "value": 3 },
+          { "label": "Small", "value": 4 },
+          { "label": "Extra small", "value": 5 }
```

Comment:
I see the use of the "value" prop as a means to identify the correct styling options...
```
const iconSizes = [48, 32, 24, 20, 18, 16]
const spaceSizes = [16, 12, 8, 6, 4, 2]
const fontSizes = [32, 24, 18, 14, 12, 10]
```

...however it might be worth considering specifying the sizes by name rather than by number.
Relying on an index feels a little brittle in the case in a couple of cases:
- we have multiple arrays that need to stay "in sync"
- we may wish to insert a new preset, e.g. between Large and Medium

Would the following make sense?

Taking just "gigantic" as an example...

In the manifest:
```
...
{ "label": "Gigantic", "value": "gigantic" },
... // other sizes
```

In the component:


```
const SIZE_PROPERTIES = new Map([
  ['gigantic', { icon: 48, space: 16, font: 32 }]
  ... // other sizes
])

// use the size properties
const size = SIZE_PROPERTIES.get(this.props.sizing) // will return `undefined` if key doesn't exist
if (size) {
  // do something
}

// ...or, check that we have a valid `sizing` prop
const hasNewButtonStyles = SIZE_PROPERTIES.has(this.props.sizing)
```

Reply:
Yeah that makes more sense. Can do!